catalog/ui/search-ui/standard/pom.xml

```diff
@@ -116,8 +116,9 @@
             <artifactId>platform-util</artifactId>
         </dependency>
         <dependency>
-            <groupId>ddf.distribution</groupId>
+            <groupId>ddf.platform</groupId>
             <artifactId>branding-api</artifactId>
+            <version>${project.version}</version>
```

Comment:
❓ This is a very minor thing, but couldn't the version be left off for all these dependencies and just inherit from the parent pom of `catalog/pom.xml`?

Reply:
Yes, the version isn't necessary because it is defined in the dependency management section of a parent pom.